#Timber beam 
###Values that have been given for this study
All measurements in metres

In [2]:
Length = 8.5;  Width = 7;   Gap = 5.5;

###Load on the beam due to the bricks in wall
$W_b$ is the weight due to the brick wall measured in Newtons. 9.81 is the acceleration due to gravity.
$W_b$ is calculated by working out the volume of the brick wall, multiplied by density, multiplied by gravity.

In [3]:
Wb = Gap * 3 * 0.12 * 1922 * 9.81 
Wb

37332.543600000005

###$W_w$ is the dead load from wall on G beam.
The factor of safety here for a dead load is 1.35 for a variable live load we would use 1.5

In [4]:
Ww = Wb *1.35
Ww

50398.93386000001

###The maximum moment generated by the wall is $M_{max}$ 
Standard formulae for a UDL and a point load have been used to work out the combined moment generated by the timber and the point load itself.

In [5]:
Mmax = Ww * Gap/8 + 40000*Gap/4 #
Mmax

89649.26702875001

### Calculation of the section modulus $W_{el}$ for a timber support  
$$W_{el} > \frac{M_{max}}{\sigma}$$ 
Consider the use of C30 timber which has $\sigma = 11$ ($\sigma$ is the bending stress parallel to plane $N/mm^2$, ref table 11.18 for values)

In [6]:
Wel = Mmax/11
Wel

8149.93336625

Unfortunately there is no timber with a $W_{el}$  $x-x$ section modulus bigger than 8150mm to accomodate this magnitude of stress applied to the C30 timber. However for completeness I shall continue to work out the deflection $\delta$ and check to see if it is less than  $\delta_{permissable}$ = 0.003*Gap

###Timber Deflection calculation
$$\delta = \Sigma C \frac{W \times L^3}{E\times I}$$
$E$ is the minumum modulus of elasticity from table 11.11 of C30 timber, $I$  $x-x$ is the second moment of inertia from table 11.18 highest available used for 300mmx300mm

In [7]:
E = 8200;  I = 675; 

The standard loading coefficients for each fraction term, $C$, have been taken from lectures in Week 21. For a uniformly distributed load $C = \frac{5}{384}$ for a point load $C = \frac{1}{48}$

In [8]:
delta = 5/384*Ww*Gap**3/(E*I)+ 1/48*40000*Gap**3/(E*I)
delta

0.044774546966645734

###Maximum permissible deflection of timber beam 
$$\delta_{permissible} = 0.003\times\text{Gap}$$

In [9]:
delta_permissible = 0.003*Gap
delta_permissible

0.0165

The deflection $\delta =0.04477$m is greater than the permissable amount $\delta_{permissible} = 0.0165$m; in addition, there is no timber with a section modulus, $W_{el}$ larger than 8150mm. Therefore I shall now consider the use of a steel beam.

#Steel Beam
###Calculation of the section modulus for a steel beam 
$$W_{el} > \frac{M_{max}}{\sigma}$$ 
Consider the use of a steel beam which has $\sigma = 275$ ($\sigma$ is the bending stress parallel to plane $N/mm^2$, ref table 11.18 for values)

The maximum moment generated by the dead load on steel beam $G$ is the same as the $M_{max}$ calculated for the timber beam. Since the steel beam has $\sigma = 275$ the minimum section modulus can be calculated.

In [10]:
steel_Wel = Mmax/275
steel_Wel

325.99733465

The section modulus suitable for this a steel beam of size 305x102x28 (section modulus 348) 
Note: The size 254x146x31 (section modulus 351) is also acceptable; however, calculations from here onwards will be for 305x102x28 (section modulus 348) which is lowest but safe value of $W_{el}$

###Maximum permissible deflection of steel beam
$$\delta_{permissible} = \frac{Gap}{360}$$

In [11]:
Gap/360

0.015277777777777777

To work out the deflection of the steel beam, values of $I$ inertia (second moment of area $x-x$) and $E$ the minumum modulus of elasticity must be used. For the steel beam 305x102x28 $I  = 5366$ with minimum modulus of elasticity of $E = 210000$.

The deflection due to live load is zero as there is only a dead load of the brick wall and point load at $A$.


###Cambered steel beams have zero deflection due to dead loading
I am not sure what to do here because in the notes for week 23,there is no $\Sigma$ on the $C\frac{W \times L^3}{E\times I}$ formula for the steel beam deflection formula. 

Cambered steel beams do not require deflection calculations due for their dead loads because the beam's camber compensates for the dead load deflection and effectively the dead load levels out the beam. 

Hence in actual fact the deflection yields $0$. 

For the sake of demonstrating understanding I have scaled the steel inertia measurement to units of $10^6mm^4$ to show the use of correct loading constants when working out deflection.   

In [12]:
steel_E =210000;     steel_I =5366/100;
steel_delta = 5/384*Ww*Gap**3/(steel_E*steel_I)+ 1/48*40000*Gap**3/(steel_E*steel_I)
steel_delta

0.021992715817438203

**As previously stated the actual deflection due to dead load would be $0$m** the above formula was used to demonstrate my ability to use the formula by selecting the correct loading constants. 

Moreover, not only is the dead load deflection zero, but the total deflection value is zero because there is no live load (variable loading on the beam) therefore $$\delta_{steel} = 0\text{m}$$ and thus this is below the required $\delta_{permissible}$ value of $0.015$m. So we may use the 305x102x28 size steel beam safely.

##Timber joists analysis

The total load carried by each joist is $W_j$ where 
$$W_j = \frac{L}{2}\times \text{spacing (m)} \times \text{load(N)} \times \text{safety factor}$$

In [13]:
Wj = 0.5*Length*0.4*1.2*10**3*1.35
Wj

2754.0

Since the moment capacity is $M_{max} = \sigma W_{el}$ we have have that $$\sigma W_{el} = \frac{L}{2}\times W_j \times\frac{1}{8}$$ rearranging this formula gives $$W_{el} = \frac{L\times W_j}{16\sigma}$$

Let us examine the possibility of using C24 timber with $\sigma= 7.5$

In [14]:
joist_Wel = Length * Wj/(16*7.5)
joist_Wel

195.075

By looking at table 11.18 it can be seen that C24 timber size 75x150 has a section modulus of 281 (>195.075) which means this choise is safe for use. 
However we must now check for deflection caused by the use of C24 75x150.
###Timber joist deflection calculation for C24 75x150
$$\delta = \Sigma C\frac{W \times Span^3}{E\times I}$$
The load is uniformly distributed over the joist so $C = \frac{5}{384}$, thus $$\delta = \frac{5}{384}\times\frac{W_j \times (\frac{L}{2})^3}{E\times I}$$
Maximum permissible delta is $$\delta_{permissible} = 0.003\times\frac{L}{2}$$

In [30]:
delta_permissible = 0.003*Length/2
delta_permissible

0.012750000000000001

For joist C24 $E = 7200$ see table 11.11, for size 75x150 $I_{x-x} = 21.1$ see table 11.18

In [35]:
delta_joistC24 = 5/384*Wj*(Length/2)**3/(7200*21.1)
delta_joistC24

0.01811984816998667

The above calculation shows us that timber C24 of size 75x150 is not safe for use because there is too much deflection $$\delta > \delta_{permissible}$$ i.e. 0.01812 > 0.01275 

Here we have two options, to find a suitable timber joist. 

We can choose a higher grade timber e.g. C30 and then repeat the analysis using the correct values for $E$ and $\sigma$ i.e. $E = 8200$ and $\sigma=11.0$ . Thereafter we would need to choose a size of beam that has a section modulus greater than the newly calculated $W_{el}$ and repeat the deflection check.

It is simpler however to resize our C24 beam (thus keeping $E = 7200$) and finding a higher inertia value of $I$.
For a C24 timber of size 75x175 $I=33.5$.

In [37]:
delta_joistC24 = 5/384*Wj*(Length/2)**3/(7200*33.5)
delta_joistC24

0.011412799892140859

This shows us that the deflection is now less than  $\delta_{permissible}$ i.e. 0.01141 < 0.01275 and so we can safely use a C24 joist of size 75x175 which has ample section modulus 383 (>195) and a small enough deflection of 0.01141 (< 0.01275).  